### ISSUE TITLE
ILIKE operator

### ISSUE THREAD

**Issue by rbayley** [tailflow/laravel-orion#57]
Hi,

Is it possible to include in the list of operators ILIKE.
The key word ILIKE can be used instead of LIKE to make the match **case-insensitive**. This is not in the SQL standard but is a PostgreSQL extension.

**Comment by alexzarbn** [comment]
Hi @rbayley,

Yes, planned it for the next feature release.

**Comment by rbayley** [comment]
Hi Folks,

Thanks for accepting this proposal.
Postgresql uses [specialized operators](https://www.postgresql.org/docs/13/ltree.html#id-1.11.7.30.6) for Ltree fields.
Is it possible to just leave this value without validation in order to use any DB specific feature?

warm regards!

**Comment by alexzarbn** [comment]
Hi @rbayley,

I would suggest to implement a version of `ParamsValidator` class that is specific to your app needs, then simply bind it to the container:

```php
$this->app->bind(Orion\Contracts\ParamsValidator::class, '<your custom params validator class>');
```

**Comment by rbayley** [comment]
Hi @alexzarbn,

Thanks for your suggestion. Will try it this way.
Although why is it important to **validate Filters**? 
Why don't just leave without validation?

**Comment by alexzarbn** [comment]
@rbayley,

For better user-feedback when consuming an API. If an end-user passes something invalid as a filter and the app runs in production, they would get a generic 500 error response. Having validation ensures a proper feedback from the API in such cases.

**Comment by alexzarbn** [comment]
The feature is now available as part of the [v2.0 release](https://github.com/tailflow/laravel-orion/releases/tag/2.0.0) 🎉
